script/cileaks.sh

```diff
@@ -9,5 +9,5 @@ then
 fi
 
 if [ -n "$VALGRIND" -a -e "$(which valgrind)" ]; then
-	valgrind --leak-check=full --show-reachable=yes --error-exitcode=125 --suppressions=./libgit2_clar.supp _build/libgit2_clar -ionline
+	valgrind --leak-check=full --show-reachable=yes --error-exitcode=125 --suppressions=./libgit2_clar.supp $@ _build/libgit2_clar -ionline
```

Comment:
This change looks unintentional?

Reply:
It's intentional, the purpose is that you can `VALGRIND=1 ./script/cileaks.sh -stestcase` in a VM and get just that valgrinded (note that I actually changed the location of the `%@`, this one is outdated).